atlasdb-cli/src/main/java/com/palantir/atlasdb/cli/AtlasCli.java

```diff
@@ -45,7 +44,6 @@ private AtlasCli() {}
                 .withCommand(Help.class)
                 .withCommand(SweepCommand.class)
                 .withCommand(KvsMigrationCommand.class)
-                .withCommand(CleanCassLocksStateCommand.class)
```

Comment:
Let me know if you have details :)

Reply:
Unfortunately, probably can't remove the code for cleaning up the lock state until we know that everyone is on a newer version of Atlas such that it can no longer occur.